.github/workflows/test-check.yaml

```diff
@@ -140,7 +140,7 @@ jobs:
       - name: "🔬 Running onnx tests"
         run: make test TARGETS=onnx
   pytorch-tests:
-    runs-on: aws-avx2-64G
```

Comment:
this runner seems to running fine with our other tests so it may no longer be an issue but if we can get away with just using an ubuntu image from github, that would be more ideal (given none of the tests require VPN)

Reply:
Could we potentially check this before landing?

Reply:
Yup, just reverted this change, lets wait and see if it runs on the aws runner